scripts/src/main/resources/scripts/command/eclipse

```diff
@@ -254,6 +258,42 @@ function doImportEclipse() {
   "${ECLIPSE_HOME}/eclipse" -nosplash -vm "${ECLIPSE_JAVA_HOME}/bin/java" -application org.eclipse.ant.core.antRunner -buildfile "${DEVON_IDE_HOME}/scripts/lib_script/import.xml" -data "${WORKSPACE_PATH}" -keyring ~/.eclipse/.keyring -vm "${ECLIPSE_JAVA_HOME}/bin/java" -vmargs -DdevonImportPath="${importPath}" -DdevonImportWorkingSet="${importWorkingSets}" ${ECLIPSE_VMARGS} 
 }
 
+# $1: URL
+# $2: name
+# $3: optional dest
+function doMirrorUpdateSite() {
+  if [ -z "${2}" ]
+  then
+    doFail "Parameter <URL> or <name> is missing."
+  fi
+  local dest="${ECLIPSE_HOME}/update-sites/"
+  local timestamp
+  timestamp=$(date +'%m-%d-%Y')
+  local eclipsecli="${ECLIPSE_HOME}/eclipse"
+  if [ -n "${3}" ]
+  then
+    dest="${3}"
+  fi
+  if doIsWindows
+  then
+    eclipsecli="${ECLIPSE_HOME}/eclipsec.exe"
```

Comment:
why dont you just call `eclipse` and keep this OS independent?

Reply:
`Eclipsec.exe` is the console executable version of eclipse.exe for Windows. Other OS's use `eclipse`. It was recommended in the docs to do it this way.

Reply:
Which "docs"? You can just call `eclipse` without any path and it works on any OS.